tests/test_template.py

```diff
@@ -38,3 +40,18 @@ def test_find_shapes_from_one_slide(template):
 def test_find_shapes_index_out_of_range(template):
     with pytest.raises(IndexError):
         shapes = template._find_shapes(0, 'logo')
+
+
+def test_save_pdf(template):
+    try:
+        subprocess.run(['libreoffice', '--version'],
+                       stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)  # check if libreoffice is installed
```

Comment:
Similar to the comment in the `Template::save_pdf()` implementation, I would urge you to move the `except` clause here for an early return.